src/assets/sass/protocol/components/_call-out.scss

```diff
@@ -0,0 +1,219 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+/* -------------------------------------------------------------------------- */
+// Call Out component
+
+.mzp-c-call-out {
+    background-color: $color-gray-20;
+    text-align: center;
+
+    &.mzp-t-dark {
+        background-color: $color-gray-90;
+        color: $color-white;
+
+        .mzp-c-call-out-desc {
+            color: $color-gray-30;
+        }
+    }
+
+    .mzp-c-call-out-title {
+        @include text-display-md;
+    }
+
+    .mzp-c-call-out-desc {
+        @include text-body-lg;
+        margin-bottom: 0;
+    }
+
+    .mzp-c-button-download {
+        margin: $margin-lg 0 0;
+    }
+
+    &.mzp-t-product-firefox ,
+    &.mzp-t-product-beta,
+    &.mzp-t-product-developer,
+    &.mzp-t-product-nightly,
+    &.mzp-t-product-focus {
+        .mzp-c-call-out-title {
+            @include background-size(64px 64px);
+            background-position: top center;
+            background-repeat: no-repeat;
+            padding: (64px + $padding-lg) 0 0 0;
+        }
+    }
+
+    &.mzp-t-product-firefox .mzp-c-call-out-title {
+        @include at2x('#{$image-path}/logos/firefox/firefox.png', 64px, 64px);
+    }
+
+    &.mzp-t-product-beta .mzp-c-call-out-title {
```

Comment:
So this line would become something like: `&.mzp-t-product-firefox.beta .mzp-c-call-out-title {`